FlowCrypt/Controllers/Inbox/InboxViewController.swift

```diff
@@ -482,7 +487,16 @@ extension InboxViewController: ASTableDataSource, ASTableDelegate {
                 node.accessibilityIdentifier = "aid-inbox-idle-node"
                 return node
             case .fetched, .refresh:
-                guard let input = self.inboxInput[safe: indexPath.row] else {
+                var rowNumber = indexPath.row
+                if self.shouldShowEmptyView {
+                    if indexPath.row == 0 {
+                        return EmptyFolerCellNode(path: self.viewModel.path, emptyFolder: {
```

Comment:
@ioanmo226 Unfortunately it seems like there is no built-in way to empty trash folder. 

It'll be better to use `users.messages.batchDelete` instead of `users.threads.delete`, as some threads can contain messages which were not moved to trash.

I found code implementation for similar task - https://medium.com/front-end-weekly/how-i-cleaned-my-gmail-using-node-js-and-gmail-api-8de34f45970d, maybe it'll be useful for you.